tasks/populate-db.mjs

```diff
@@ -33,8 +33,8 @@ const cleanDB = cssdb.map(
 	 }) => properties
 );
 
-const newCSSDB = JSON.stringify(cleanDB, null, 2);
-const esmCSSDB = `export default ${newCSSDB}`;
+const newCSSDB = `${JSON.stringify(cleanDB, null, 2)}\n`;
+const esmCSSDB = `export default ${newCSSDB}\n`;
```

Comment:
```suggestion
const esmCSSDB = `export default ${newCSSDB}`;
```

Tiny detail :)
Final newline is already above, so not needed here.

Reply:
🙈